bob-apps/Cargo.toml

```diff
@@ -47,8 +47,9 @@ lazy_static = "1.4"
 log = "0.4"
 log4rs = "1.2"
 metrics = { version = "0.17", features = ["std"] }
+network-interface = "0.1.2"
+prost = "0.9"
```

Comment:
Incorrect version downgrade. Revert this change and change below (where `prost = "0.11"` was removed)